web/static/actor/index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Actor Info</title>
+    <script type="module" src="/ux/curio-ux.mjs"></script>
+    <script type="module" src="/actor-summary.mjs"></script>
+</head>
+<body style="visibility:hidden" data-bs-theme="dark">
+<curio-ux>
+    <actor-info></actor-info>
+</curio-ux>
+<script>
+    import { LitElement, html, css } from 'https://cdn.jsdelivr.net/gh/lit/dist@3/all/lit-all.min.js';
+    import RPCCall from '/lib/jsonrpc.mjs';
+    customElements.define('actor-info',class Actor extends LitElement {
+    constructor() {
+        super();
+        this.loadData();
+    }
+
+    async loadData() {
+        this.data = await RPCCall('ActorSummary', true);
```

Comment:
RPCCall args must be an array
```suggestion
        this.data = await RPCCall('ActorSummary', [true]);
```

(here and the change in actor-summary.mjs)

Reply:
Also, this page is meant to be per-actor, not an overview, the url is like `/actor/?id=t0116147`; Should likely just use a separate per-actor RPC method